Cargo.toml

```diff
@@ -1,36 +1,39 @@
-[package]
-name = "bevy_app_compute"
-version = "0.10.3"
-authors = ["Kjolnyr <kjolnyr@protonmail.com>"]
-edition = "2021"
-description = "App compute plugin for Bevy"
-repository = "https://github.com/Kjolnyr/bevy_app_compute"
-homepage = "https://github.com/Kjolnyr/bevy_app_compute"
-documentation = "https://docs.rs/bevy_app_compute"
-license = "MIT OR Apache-2.0"
-readme = "README.md"
-categories = ["game-development"]
-
-
-[dependencies]
-bevy = "0.10.1"
-parking_lot = "0.12.1"
-wgpu = "0.15.1"
-codespan-reporting = "0.11.1"
-futures-lite = "1.13.0"
-bytemuck = "1.13.1"
-
-[dev-dependencies]
-rand = "0.8.5"
-
-[[example]]
-name = "simple"
-
-[[example]]
-name = "multi_pass"
-
-[[example]]
-name = "one_shot"
-
-[[example]]
-name = "boids"
\ No newline at end of file
+[package]
+name = "bevy_app_compute"
+version = "0.10.5"
```

Comment:
#4 bumps the version to `0.11.0` -- more generally I was wondering whether this PR be should be based on #4 and release a version of the library that works with bevy 0.11 before merging this PR. Just academic, I'm already on bevy 0.12

Reply:
You are absolutely right, I don't think it should be merged before the 0.11 PR is merged, I based the code in this PR on the 0.11 PR so there shouldn't be major conflicts and even if there are I'm committed to make this crate work on bevy 0.12 because I need something like this for my game :)

Reply:
> #4 bumps the version to `0.11.0` -- more generally I was wondering whether this PR be should be based on #4 and release a version of the library that works with bevy 0.11 before merging this PR. Just academic, I'm already on bevy 0.12

I'll leave it to the package author how to handle the versioning, but I agree that there should be two releases for bevy 0.11 and 0.12 separately.